src/ProxyClient/Cloudflare.php

```diff
@@ -169,4 +167,14 @@ protected function configureOptions()
 
         return $resolver;
     }
+
+    private function json_encode(array $data): string
```

Comment:
@dbu a minor point but should this method be named `jsonEncode()` to fit in with naming format used elsewhere?

Reply:
my thinking was that it then looks like the native function, to be clear it is just a wrapper around that. in our applications, we use https://github.com/thecodingmachine/safe to wrap around all the old school php functions to have them throw exceptions. but i don't want to introduce that here as a dependency.

ok to keep it with underscore, or do you find it confusing that way?

Reply:
That makes sense, please do leave as is :)